lib/config.js

```diff
@@ -159,15 +170,49 @@ function sort (a, b) {
 }
 
 function publicVar (k) {
-  return !(k.charAt(0) === '_' ||
-           k.indexOf(':_') !== -1 ||
-           types[k] !== types[k])
```

Comment:
`types[k] !== types[k]` is a hold over from when the type for `argv` was `NaN`. Cute trick to exclude `argv` but was removed a long long time ago, so this has done nothing for like 4 years except confuse.